data/hai/hai missions.txt

```diff
@@ -407,15 +407,9 @@ mission "Unfettered Jump Drive 2"
 mission "Unfettered Jump Drive 3"
 	minor
 	landing
```

Comment:
This mission doesn't fail fast enough, and shows up in the job board before jumping/reloading. (We could readd the description instead of making this mission invisible, but that still causes problems with the mission vanishing for arbitrary reasons.)
```suggestion
	landing
	invisible
```